code/datums/components/riding.dm

```diff
@@ -64,7 +65,10 @@
 /datum/component/riding/proc/ride_check(mob/living/M)
 	var/atom/movable/AM = parent
 	var/mob/AMM = AM
-	if((ride_check_rider_restrained && M.restrained(TRUE)) || (ride_check_rider_incapacitated && M.incapacitated(FALSE, TRUE)) || (ride_check_ridden_incapacitated && istype(AMM) && AMM.incapacitated(FALSE, TRUE)))
+	var/kick_us_off
+	if((ride_check_rider_restrained && M.restrained(TRUE)) || (ride_check_rider_incapacitated && M.incapacitated(TRUE, TRUE)))

```

Comment:
`incapacitated(FALSE, TRUE)` means it was checking for cuffs. this check should've only been performed via ride_check_rider_restrained. 

Nothing used this yet, so it it went unnoticed.